src/main/java/com/google/api/generator/spring/utils/Utils.java

```diff
@@ -81,6 +83,14 @@ public static String getSpringPropertyPrefix(String packageName, String serviceN
     return packageName + "." + CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_HYPHEN, serviceName);
   }
 
+  public static List<Method> getMethodsForRetryConfiguration(Service service) {
+    // Returns list of methods with retry configuration support
+    // This currently excludes streaming and LRO methods
+    return service.methods().stream()
```

Comment:
Just a note that we need to revisit this logic before GA, as we discussed, BIDI and Client streaming do not support retry, but Server streaming and LRO should support it.

Reply:
Noted - thank you!